docs/it/week03/03-2.md

```diff
@@ -59,7 +59,7 @@ L'anno successivo, si fecero alcuni cambiamenti: venne introdotto uno strato di
 
 Il seguente è un esempio di una rete con un solo strato convoluzionale:
 1. Si consideri un input con dimensione *32*$\times$*32*
-2. Lo strato convoluzionale passa un filtro (_kernel_) di dimensione *5*$\times$*5* con _stride_ 1 sull'immagine, creando una mappa delle caratteristiche *28*$\times$*28*
+2. Lo strato convoluzionale passa un _kernel_ di dimensione *5*$\times$*5* con _stride_ 1 sull'immagine, creando una mappa delle caratteristiche *28*$\times$*28*
```

Comment:
Why are you replacing every instance of *filtro* with *kernel*?

Reply:
Fixed it